SDL/SDLMain.mm

```diff
@@ -374,6 +375,11 @@ int main (int argc, char **argv)
 
     PROFILE_INIT();
 
+	// Ignore sigpipe.
+	if (signal(SIGPIPE, SIG_IGN) == SIG_ERR) {
+		perror("Unable to ignore SIGPIPE");
+	}
+
 #if SDL_USE_NIB_FILE
```

Comment:
NItpick: Weird indentation here

Reply:
Ah, didn't notice it was using spaces.  Updated .editorconfig too.

-[Unknown]